app/views/pages/start.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= content_for :banner do %>
+  <%= render_component StartPageBanner::View.new %>
+<% end %>
+
+<div class="govuk-grid-row govuk-main-wrapper--auto-spacing">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">Use this service to:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>upload Assessment Only (AO) candidate data</li>
+      <li>get a Teacher Reference Number (TRN) for AO candidates</li>
+      <li>recommend candidates for Qualified Teacher Status (QTS)</li>
+      <li>check and update your AO candidate data</li>
+    </ul>
+
+    <p class="govuk-body">
+      If you are a Higher Education Institute (HEI), you must submit your AO data using the Register trainee teacher
+      service. You do not submit it via the Higher Education Statistics Agency (HESA).
```

Comment:
```suggestion
      If you are a Higher Education Institute (HEI), you must submit your AO data using the Register trainee teachers
      service. You do not submit it via the Higher Education Statistics Agency (HESA).
```

![image](https://user-images.githubusercontent.com/470137/94283345-ca996480-ff48-11ea-8c97-ccecfc426c36.png)

Are we refering back to ourselves... are we plural or singular :thinking: